src/components/tasks/CreateTask.component.scss

```diff
@@ -0,0 +1,12 @@
+@import "src/assets/styles/_variables.scss";
```

Comment:
Is it possible to config this variables file globally instead of importing into all scss manually?